src/tasks/processIssueComment/utils/parse-comment/index.js

```diff
@@ -77,6 +79,7 @@ const contributionTypeMultiWordMapping = {
     'fund finding': 'fundingFinding',
     'funding finding': 'fundingFinding',
     'user testing': 'userTesting',
+    'business development': 'business',
```

Comment:
Awesome! Do we need to bump the all-contributors-cli version in the bot?

Reply:
It would better than not doing it especially considering that the [respective changes](https://github.com/all-contributors/all-contributors-cli/commit/48fecd3ca779d44558000a4085dc36897fdc3ca8) are in more [recent version](https://github.com/all-contributors/all-contributors-cli/releases/tag/v5.11.0).

Reply:
@jakebolam Sounds good.

Reply:
The other dependencies were upgrade in #128.